game-server/data/handlers/ai/instance/empyreanCrucible/KingConsierdAI.java

```diff
@@ -9,20 +9,18 @@
 import com.aionemu.gameserver.controllers.attack.AggroInfo;
 import com.aionemu.gameserver.model.gameobjects.Creature;
 import com.aionemu.gameserver.model.gameobjects.Npc;
-import com.aionemu.gameserver.skillengine.SkillEngine;
 import com.aionemu.gameserver.utils.ThreadPoolManager;
-import com.aionemu.gameserver.world.WorldPosition;
 
 import ai.AggressiveNpcAI;
 
 /**
- * @author Luzien
+ * @author Luzien, w4terbomb
  */
 @AIName("king_consierd")
 public class KingConsierdAI extends AggressiveNpcAI implements HpPhases.PhaseHandler {
 
 	private final HpPhases hpPhases = new HpPhases(75, 25);
-	private AtomicBoolean isHome = new AtomicBoolean(true);
+	private final AtomicBoolean isHome = new AtomicBoolean(true);
```

Comment:
This field can be removed by instantiating hpPhases via `new HpPhases(100, 75, 25)` and starting the tasks from `handleHpPhase` in `case 100`.